Dockerfile

```diff
@@ -3,7 +3,7 @@ WORKDIR /opt/app/
 COPY package.json /opt/app/
 RUN apk --no-cache add --virtual native-deps \
   g++ make python && \
-  npm install --quiet --production && \
+  npm install --no-optional --quiet --production && \
```

Comment:
this is to force `fsevents` to install even though it's an optional dependency, otherwise it leads to a bunch of ugly warnings.